spec/features/user_spec.rb

```diff
@@ -117,4 +117,21 @@
     expect(deactive_links.count).to eq(4)
     expect(activated_links.count).to eq(0)
   end
+
+  scenario 'should allow an admin to see User information page with loan history' do
+    sign_in admin
+    visit loan_listing_url
+    loan = loans(:outstanding)
+    loan.member = member
+    loan.save!
+
+    click_link ("#{member.first_name } #{member.last_name}"), :match => :first
+
+    expect(page).to have_content(member.email)
+    expect(page).to have_content(member.phone_number)
+    expect(page).to have_content(member.created_at)
+    expect(page).to have_content(member.role)
+    expect(page).to have_content("Active")
+    expect(page).to have_content(loan.carrier.name)
```

Comment:
It seems as if the loan history isn't showing up here so I've done something wrong in the setup here.  I'm not sure what though, @mark100net do you see something here that might help me?

Reply:
This is fixed now.